loadConn.go

```diff
@@ -21,6 +21,7 @@ type IKEConf struct {
 	RemoteAuth  AuthConf               `json:"remote"`
 	Pools       []string               `json:"pools,omitempty"`
 	Children    map[string]ChildSAConf `json:"children"`
+	Mobike      string                 `json:"mobike"`
```

Comment:
I suspect you just need to add ,omitempty to make the tests pass in the absence of this flag being set.

Reply:
I'll go ahead and push a change with this very quickly